src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -740,13 +740,16 @@ try
 CATCH_LOG()
 
 // Method Description:
-// - Sets the visibility of the text cursor.
+// - Sets the cursor to be currently on. On/Off is tracked independantly of

```

Comment:
independently

Reply:
asfdgkajenfklaerba classic me

Reply:
Man, go read that spell check PR #4799 so we can make a computer catch these things for you.